docs/Overview.rst

```diff
@@ -28,8 +28,8 @@ The user is asked to enter the required data through a web interface. In develop
 
 4.	Meteorological data, which is related to the components that generate electricity by harnessing an existing source of energy that is weather- and time-dependent (e.g., solar and wind power).
 
-This set of input data is then translated to a linear programming problem, also known as a constrained optimization problem. The MVS is based on the oemof python library that describes the problem by specifying the objective function, the decision variables and the constraints. The aim is to minimize the production costs by determining the generating units' optimal output, which meets the total demand.
-The simulation outputs are also separated into categories. The economic results, such as the levelized cost of electricity or heat or the net present value of the projected investments, are the ones used for financial evaluation and decision making. The technical results also showcase several important values, such as the optimized capacities and dispatch of each asset. Lastly, the system’s environmental contribution is shown through the value of CO2 emissions. 
+This set of input data is then translated to a linear programming problem, also known as a constrained optimization problem. The MVS is based on the oemof-solph python library that describes the problem by specifying the objective function, the decision variables and the bounds and constraints. The goal is to (1) minimize the production costs by determining the generating units' optimal output, which meets the total demand, and (2) otpimize near-future investments in generation and storage assets with the least possible cost of energy.
+The simulation outputs are also separated into categories. The economic results, such as the levelized cost of electricity or heat or the net present value of the projected investments, are the ones used for financial evaluation and decision making. The technical results also showcase several important values, such as the optimized capacities and dispatch of each asset. Lastly, the system’s environmental contribution is shown through the value of CO2 emissions.
```

Comment:
"...are the ones used for financial evaluation and decision making." -> I would set decision making at the end of the paragraph, otherwise the techno-environmental parameters are just additional but completely disregardable indicators, as they are not included in the *real* decision making process.

Reply:
okay I can change that, but I was just trying to follow the order of appearance in the flowchart. is it still fine then?

Reply:
From my point of view, this order would not change. If you put "decision making" infront of or after all the outputs (eco, techno, environ), it says that you consider all three for the decision making. You have to balance out economic benefits with techno/environ requirements etc.